dod-advana/gamechanger-web
# Task/uot 131487

## Description



- Increases search matches on individual keywords (up to a certain amount due to performance) on titles. i.e. not every word in a search has to be in for a title match
- pagerank reintroduced as a rescore, this increases relevancy of highly referenced documents helping important docs ranking
- misc: bugfix with forward slashes in search, refactoring, intelligent search shows more characters now
## !vibez



## Related Issue/Ticket



[UOT-133265](https://jira.di2e.net/browse/UOT-133265)
[UOT-135007](https://jira.di2e.net/browse/UOT-135007)
## Types of changes

## Smoke testing instructions



- Run searches on titles with a `/`
- Run searches on a word thats apart of a title with a word that isnt. i.e. `directive 3118.03`
- run searches on titles that should show up above noise that are common like `arbitration` or `commerce and trade`
## Checklist:
